report/report-server/src/main/kotlin/com/testerum/report_server/controller/ReportController.kt

```diff
@@ -0,0 +1,19 @@
+package com.testerum.report_server.controller
+
+import com.testerum.report_server.service.ReportService
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/report/v1", produces = ["application/json"])
+class ReportController(private val reportService: ReportService) {
+
+    @PostMapping()
```

Comment:
minor: remove the unnecessary parenthesis:
```suggestion
    @PostMapping
```